plaso/lib/line_reader_file.py

```diff
@@ -165,3 +165,39 @@ def tell(self):
       int: current offset into the file-like object.
     """
     return self._current_offset
+
+
+class BinaryDSVReader(object):
+  """Basic reader for delimiter separated text files of unknown encoding.
+
+  This is used for reading data from text files where the content is unknown, or
+  possibly mixed.
+  """
+
+  def __init__(self, binary_line_reader, delimiter):
+    """Initializes the delimited separated values reader.
+
+    Args:
+      binary_line_reader (BinaryLineReader): a binary file reader
+      delimiter (bytes): field delimiter.
+    """
+    super(BinaryDSVReader, self).__init__()
+    self._line_reader = binary_line_reader
+    self._delimiter = delimiter
+
+  def __iter__(self):
+    """Returns separated values.
+
+    Yields:
+      list(bytes): lines of encoded bytes.
+    """
```

Comment:
Add a Raises section to the docstring

Reply:
I'm not sure what you want here - this function doesn't raise directly. I've added a Raises section to readline() in the line reader as that's where exceptions are actually raised.

Reply:
ack, could be an oversight due to the github review UI